ui/lib/components/accordion/accordion.dart

```diff
@@ -31,7 +31,14 @@ class AccordionItem {
   DivElement _headerWrapper;
   DivElement _bodyWrapper;
 
-  AccordionItem(this.id, this.headerElement, this.bodyElement, this._isOpen, {String dataId}) {
+  bool get isOpen => _isOpen;
+
+  void set id (String newID) {
+    _id = newID;
```

Comment:
nit:
```suggestion
  void set id(String value) {
    _id = value;
```
(for setters, because the syntax for using them is a bit different to passing arguments to normal functions, we can use `value` as the name as an easy way of not having to think up another name, and it won't be visible to the user)